cli/script/command-executor.ts

```diff
@@ -178,6 +178,24 @@ function appList(command: cli.IAppListCommand): Promise<void> {
         });
 }
 
+function checkIfAppExists(appName: string): Promise<App> {
+    log(chalk.cyan(`Checking if app with the name "${appName}" exists...\n`));
+    return Promise<App>((resolve, reject) => {
+        sdk.getApps()
```

Comment:
You can just return `sdk.getApps() ...` directly instead of wrapping in a promise, as these methods return a promise. Below, you can remove the `if (app)` branch, because if nothing is returned, the promise resolves as success, and instead of `reject(error)`, do `throw new Error(...)` which rejects the promise and matches what we are doing elsewhere.